pkgs/tools/misc/gay/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, python39Packages, fetchFromGitHub, runCommand, callPackage }:
+python39Packages.buildPythonPackage rec {
+  name = "gay";
+  version = "1.2.8";
+
+  src = fetchFromGitHub {
+    owner = "ms-jpq";
+    repo = "gay";
+    rev = "1e3e96815c68214533a925c86a52b0acf832a359";
```

Comment:
Why are we not using the tag?

Reply:
Last I checked, the repo doesn't have any tags.

Reply:
Can you add that explanation as a comment? In the meantime I’ve opened an issue: https://github.com/ms-jpq/gay/issues/12